content/Concepts/terminology.mdx

```diff
@@ -0,0 +1,126 @@
+---
+slug: terminology
+title: Configu's Terminology
+---
+
+Here is a dictionary of some key concepts in Configu:
+
+<div class="table-wrapper">
+
+  <table>
+    <thead>
+      <tr>
+        <th>Concept</th>
+        <th>Explanation</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <td>
+          <strong>Config</strong>
+        </td>
+        <td>
+          A Config is a generic representation of a software configuration. It is the data model that Configu uses to store and fetch configuration data. A Config consists of four major properties:
+          - key: The name of the configuration as it appears in the code and in the ConfigSchema.
+          - schema: The unique identifier of the ConfigSchema that the configuration has been declared on.
+          - value: The literal value of the configuration in the ConfigSet.
+          - set: The path of the ConfigSet that the value belongs to.
+          <a href="https://github.com/configu/configu/blob/main/types/Config.ts">See Config decleration.</a>
+        </td>
+      </tr>
+
+      <tr>
+        <td>
+          <strong>Cfgu</strong>
+        </td>
+        <td>
+          A Cfgu is a generic declaration of a Config, that specifies information about its type, constrains and other characteristics.
+          <a href="https://github.com/configu/configu/blob/main/types/Cfgu.ts">See Cfgu decleration.</a>
+        </td>
+      </tr>
+
+      <tr>
+        <td>
+          <strong>ConfigSchema</strong>
+        </td>
+        <td>
+          A ConfigSchema is a file stored as \[Config.schema\].cfgu.\[json|yaml\] that contains binding records between a unique \[Config.key\] and its Cfgu declaration.
+          <a href="/docs/config-schema">Read more on ConfigSchema</a>
```

Comment:
The two links seem like one and the Read more on... is a broken link in all of them

![image](https://user-images.githubusercontent.com/50705727/209942768-7c92fb58-5da1-493d-9e82-c58ef1263ba4.png)